docroot/themes/humsci/su_humsci_theme/scss/colors.scss

```diff
@@ -0,0 +1,94 @@
+@charset "UTF-8";
+
+$color-bright-red: rgb(177, 4, 14) !default;
```

Comment:
I understand this is how the color module works but I wish it wasn't so. It would be a better step if you could change the way that the color module replaces colors in the stylesheet to instead change a color hex with another color hex, to replacements with css variables:

https://caniuse.com/#search=css%20variables
https://medium.freecodecamp.org/everything-you-need-to-know-about-css-variables-c74d922ea855

That way you could avoid having to change the color values by 1 for each element.

Reply:
that might be out of scope for this. That would involve completely altering the entire functionality of the form and the submit handler. essentially replacing about 3/4 of the color module